CI/espresso_tests/unit_tests/test_integration.py

```diff
@@ -0,0 +1,227 @@
+import multiprocessing
+import tempfile
+import time
+import traceback
+import unittest as ut
+
+import numpy as np
+import pint
+
+import swarmrl.engine.espresso as espresso
+from swarmrl.models import dummy_models
+from swarmrl.utils import utils
+
+
+class Process(multiprocessing.Process):
+    """
+    Process class for use in multi simulation testing.
+    """
+
+    def __init__(self, *args, **kwargs):
+        """
+        Multiprocessing class constructor.
+        """
+        multiprocessing.Process.__init__(self, *args, **kwargs)
```

Comment:
This test is wild. but if it works it works

Reply:
It is based on a hint from Sam. I could integrate over all the process calls from one function and have several simulation classes as well but the Simulations are similar so that maybe it is ok to stick to this. (I try to have rather little  code duplication but still readable code.) This test structure is also implemented in the visualization branch.